config/routes.rb

```diff
@@ -5,12 +5,17 @@
              skip: [:sessions]
 
   devise_scope :user do
-    get 'wp-admin', to: 'devise/sessions#new', as: :new_user_session
-    post 'wp-admin', to: 'devise/sessions#create', as: :user_session
-    delete 'logout', to: 'devise/sessions#destroy', as: :destroy_user_session
+    get 'sign-in', to: 'devise/sessions#new', as: :new_user_session
   end
 
   namespace :account do
+    resources :issues, only: [:index] do
+      member do
+        patch :approve
+        delete :destroy
```

Comment:
this action is by default provided by REST, remove it from routes, but add to "only" (above)